README.md

```diff
@@ -149,8 +149,9 @@ podman run quay.io/redhat_msi/openshift-cli-installer \
     --registry-config-file=registry-config.json \
     --s3-bucket-name=openshift-cli-installer \
     --s3-bucket-path=install-folders \
-    --cluster 'name=ipi1;base_domain=aws.interop.ccitredhat.com;platform=aws;region=us-east-2;version=4.14.0-ec.2;worker_flavor=m5.xlarge'
+    --cluster 'name=ipi1;base_domain=aws.interop.ccitredhat.com;platform=aws;region=us-east-2;version=4.14.0-ec.2;worker_flavor=m5.xlarge;log-level=error'
 ```
+  * Pass `log-level=error` to cluster config if user want to hide the openshift-installer logs which contains `kubeadmin` password.
```

Comment:
I think the default should be error, if info expose passwords

Reply:
Yes, we can default to error.
And working on adding the check for log level options

Reply:
Updated the code with check @myakove